Emma/emma_libs/memoryManager.py

```diff
@@ -16,32 +16,35 @@
 along with this program.  If not, see <https://www.gnu.org/licenses/>
 """
 
-
 import os
 
 from pypiscout.SCout_Logger import Logger as sc
 # import graphviz
 
-from Emma.shared_libs.stringConstants import *                           # pylint: disable=unused-wildcard-import,wildcard-import
+from Emma.shared_libs.stringConstants import *  # pylint: disable=unused-wildcard-import,wildcard-import
```

Comment:
Keep initial indentation.
```suggestion
from Emma.shared_libs.stringConstants import *                           # pylint: disable=unused-wildcard-import,wildcard-import
```